src/utils/DockerUtil.js

```diff
@@ -263,6 +271,68 @@ var DockerUtil = {
     });
   },
 
+  fetchAllNetworks () {
+    this.client.listNetworks((err, networks) => {
+      if (err) {
+        networkActions.error(err)
+      } else {
+        networks = networks.filter(n => n.Name !== 'host').sort((n1, n2) => {
```

Comment:
When I trying to connect any container to `host` network from cli i got an error:

```
$ docker network connect host blackfire
Error response from daemon: Container cannot be disconnected from host network or connected to host network
```